packages/auth/src/mfa/assertions/totp.test.ts

```diff
@@ -208,6 +213,91 @@ describe('core/mfa/totp/assertions/TotpMultiFactorAssertionImpl', () => {
   });
 });
 
+describe('Testing signin Flow', () => {
+  let auth: TestAuth;
+  let assertion: MultiFactorAssertionImpl;
+  let totpSignInResponse: StartTotpMfaSignInResponse;
+  let session: MultiFactorSessionImpl;
+  beforeEach(async () => {
+    mockFetch.setUp();
+    auth = await testAuth();
+    session = MultiFactorSessionImpl._fromMfaPendingCredential(
+      'mfa-pending-credential'
+    );
+  });
+  afterEach(mockFetch.tearDown);
+
+  it('should finalize mfa signin for totp', async () => {
+    
+    totpSignInResponse = { 
+      verificationCode: '123456',
+      idToken: 'final-id-token',
+      refreshToken: 'refresh-token'
+     } as any;
```

Comment:
The use of `any` here will probably fail the linter

Reply:
It does not perhaps because it's a test file